packages/@o3r/localization/src/devkit/localization-devkit.interface.ts

```diff
@@ -20,3 +28,46 @@ export type AvailableLocalizationMessageContents =
   | LocalizationMessageContents
   | ConnectContentMessage
   | RequestMessagesContentMessage<LocalizationMessageDataTypes>;
+
+
+/**
+ * Contextualization devtools exposed for localization in CMS integration
+ */
+export interface LocalizationContextualizationDevtools {
+  /**
+   * Show localization keys
+   * @param value value enforced by the DevTools extension
+   */
+  showLocalizationKeys: (value?: boolean) => void;
```

Comment:
Maybe we should make all these functions asynchronous, just in case we'll need that at some point to avoid breaking changes